src/backend/access/transam/xact.c

```diff
@@ -902,6 +902,12 @@ GetCurrentCommandId(bool used)
 	return currentCommandId;
 }
 
+void
+SetCurrentCommandId(CommandId cid)
```

Comment:
how to sync currentCommandId in parallel mode ?

Reply:
currentCommandId will automatically synced in SerializeTransactionState function just like current xid.